apps/src/sites/studio/pages/levels/_standalone_video.js

```diff
@@ -0,0 +1,14 @@
+import $ from 'jquery';
+import { registerGetResult } from '@cdo/apps/code-studio/levels/codeStudioLevels';
+import { postMilestoneForPageLoad, onContinue } from '@cdo/apps/code-studio/levels/postOnLoad';
+
+$(document).ready(() => {
+  registerGetResult();
+
+  // make milestone post
+  postMilestoneForPageLoad();
+
+  // handle click on continue, in particular think about what happens if we didnt
+  // hear back
```

Comment:
Does this comment, "think about what happens if we didn't hear back," still apply?